source/_docs/scripts.markdown

```diff
@@ -604,6 +604,22 @@ This action supports nesting, however, if you find yourself using nested if-then
 actions in the `else` part, you may want to consider using
 [choose](#choose-a-group-of-actions) instead.
 
+[Template shorthand notation](https://www.home-assistant.io/docs/scripts/conditions/#template-condition-shorthand-notation) can be used with if-then. The code below is equivalent to the example above.
+
+```yaml
+script:
+  - if: "{{ is_state('zone.home', 0) }}"
```

Comment:
This feels misplaced? As in, like an oddity to mention in the middle out of the blue?
This page is not aimed at showing how conditions work.

Reply:
We have something pretty much equivalent for [Repeat-until](https://www.home-assistant.io/docs/scripts/#repeat-until) on this page, which is why I thought here was sensible